OSDOCS-19924:Update the z-stream RNs for 4.18.43 (#112561)
openshift/openshift-docs: modules/zstream-4-18-43.adoc

// Module included in the following assemblies:
//
// * release_notes/ocp-4-18-release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="zstream-4-18-43_{context}"]
= RHSA-2026:21657 - {product-title} {product-version}.43 fixed issues and security update

Issued: 03 June 2026

[role="_abstract"]
{product-title} release {product-version}.43 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2026:21657[RHSA-2026:21657] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2026:21655[RHSA-2026:21655] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.18.43 --pullspecs
----
[id="zstream-4-18-43-enhancements_{context}"]
== Enhancements

> * With this enhancement, the must-gather diagnostic tool will collect the total size of every type of resource in etcd to assist with many control plane troubleshooting scenarios. (link:https://redhat.atlassian.net/browse/OCPBUGS-86019[OCPBUGS-86109)

Reviewer: ```suggestion
* With this enhancement, the must-gather diagnostic tool will collect the total size of every type of resource in etcd to assist with many control plane troubleshooting scenarios. (link:https://redhat.atlassian.net/browse/OCPBUGS-86019[OCPBUGS-86109])
```

[id="zstream-4-18-43-known-issues_{context}"]
== Known issues

> * On certain Telecom Grandmaster (T-GM) systems, an {product-title} node reboot can trigger a change in the Global Navigation Satellite System (GNSS) device name. This issue prevents the T-GM system from achieving a locked state because the configuration points to a static, stale device path. To work around this problem, omit the `ts2phc.nmea_serialport` setting from the `ts2phcConf` section of the `PtpConfig` resource. (link:https://redhat.atlassian.net/browse/OCPBUGS-77517[OCPBUGS-77517)

Reviewer: ```suggestion
* On certain Telecom Grandmaster (T-GM) systems, an {product-title} node reboot can trigger a change in the Global Navigation Satellite System (GNSS) device name. This issue prevents the T-GM system from achieving a locked state because the configuration points to a static, stale device path. To work around this problem, omit the `ts2phc.nmea_serialport` setting from the `ts2phcConf` section of the `PtpConfig` resource. (link:https://redhat.atlassian.net/browse/OCPBUGS-77517[OCPBUGS-77517])
```

[id="zstream-4-18-43-fixed-issues_{context}"]
== Fixed issues

> * Before this update, the `Modern` Transport Layer Security (TLS) security profile did not work. With this release, support for TLS 1.3 and the proper handling of the `Modern` profile is added, which ensures that the `Modern` TLS security profile works as expected. (link:https://redhat.atlassian.net/browse/OCPBUGS-56834[OCPBUGS-56834)

Reviewer: ```suggestion
* Before this update, the `Modern` Transport Layer Security (TLS) security profile did not work. With this release, support for TLS 1.3 and the proper handling of the `Modern` profile is added, which ensures that the `Modern` TLS security profile works as expected. (link:https://redhat.atlassian.net/browse/OCPBUGS-56834[OCPBUGS-56834])
```

* Before this update, enabling the `routingViaHost: true` and `ipForwarding: Global` parameters on a {hcp} KubeVirt guest cluster caused the console and ingress Cluster Operators to become degraded. As a consequence, route health checks from within the cluster would fail, even though ingress routes remained accessible from external clients. This issue occurred due to a routing conflict when both the management cluster and the {hcp} guest cluster had the `routingViaHost: true` parameter enabled. With this release, the fix ensures that {hcp} KubeVirt guest clusters use the subnet gateway IP as the default gateway. As a result, the routing conflict is resolved and the console and ingress Cluster Operators are allowed to remain healthy when these networking features are enabled. (link:https://redhat.atlassian.net/browse/OCPBUGS-74399[OCPBUGS-74399])

> * Before this update, the `oauth` pods in the `openshift-authentication` namespaces could get stuck while rolling out changes when one of the respective nodes these pods were running on was not ready or not available. As a consequence, authentication completely stopped until the blocked rollout concluded. With this release, the pods can proceed with the rolling update even when an unhealthy node is down or unavailable (link:https://redhat.atlassian.net/browse/OCPBUGS-75897[OCPBUGS-75897])

Reviewer: I reckon there is only one `openshift-authentication` namespace, and I'm not clear on the closing phrase: an _unhealthy_ node is down or unavailable. (Which suggests that there is still a problem if a _healthy_ node is down or unavailable.)
```suggestion
* Before this update, the `oauth` pods in the `openshift-authentication` namespace could get stuck while rolling out changes when one of the respective nodes these pods were running on was not ready or not available. As a consequence, authentication completely stopped until the blocked rollout concluded. With this release, the pods can proceed with the rolling update even when a node is down or unavailable (link:https://redhat.atlassian.net/browse/OCPBUGS-75897[OCPBUGS-75897])
```
